appcues/src/main/java/com/appcues/trait/appcues/TargetElementTrait.kt

```diff
@@ -15,7 +15,7 @@ internal class TargetElementTrait(
 
         const val TYPE = "@appcues/target-element"
 
-        const val METADATA_TARGET_RECT = "targetRectangle"
+        const val METADATA_TARGET_RECT = "targetElement"
```

Comment:
most for understanding, this is not being used right now but I noticed it was referring to targetRectangle, when we get to targetElement we might want to have both in different keys, for now will leave as this to avoid any confusion

Reply:
I think we may always want "targetRectangle"? shared between anything that consumes the target-element and the target-rectangle trait. Both of those will resolve to an underlying rectangle that other things (like keyhole) will use. I don't think we'll have a separate "targetElement" metdata - just it's underlying rectangle.

Reply:
makes sense, unless we want to somehow combine both to make something very flexible. will keep an eye for that when we get to that.